<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterEditor</class>
 <widget class="QMainWindow" name="FilterEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="uiCentralWidget">
   <layout class="QVBoxLayout" name="layoutMain">
    <item>
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="DeselectableQTreeView" name="uiTree"/>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="uiHelp">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>50</horstretch>
              <verstretch>30</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Help</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons/icons.qrc">
              <normaloff>:/breeze/light/help-about.svg</normaloff>:/breeze/light/help-about.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="uiAddButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>50</horstretch>
              <verstretch>30</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Add a new node</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons/icons.qrc">
              <normaloff>:/breeze/light/list-add.svg</normaloff>:/breeze/light/list-add.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uiRemoveButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>30</horstretch>
              <verstretch>30</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Delete node</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons/icons.qrc">
              <normaloff>:/breeze/light/edit-delete.svg</normaloff>:/breeze/light/edit-delete.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uiUpButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>30</horstretch>
              <verstretch>30</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Move node up</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons/icons.qrc">
              <normaloff>:/breeze/light/go-up.svg</normaloff>:/breeze/light/go-up.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="uiDownButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>30</horstretch>
              <verstretch>30</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>30</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Move node down</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../icons/icons.qrc">
              <normaloff>:/breeze/light/go-down.svg</normaloff>:/breeze/light/go-down.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="uiMenubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="uiActionOpen"/>
    <addaction name="separator"/>
    <addaction name="uiActionSave"/>
    <addaction name="uiActionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="uiActionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="uiActionHelp"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="uiActionOpen">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/breeze/light/document-open.svg</normaloff>:/breeze/light/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="uiActionSave">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/breeze/light/document-save.svg</normaloff>:/breeze/light/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="uiActionSaveAs">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/breeze/light/document-save-as.svg</normaloff>:/breeze/light/document-save-as.svg</iconset>
   </property>
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="uiActionExit">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/breeze/light/application-exit.svg</normaloff>:/breeze/light/application-exit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="uiActionHelp">
   <property name="icon">
    <iconset resource="../icons/icons.qrc">
     <normaloff>:/breeze/light/help-about.svg</normaloff>:/breeze/light/help-about.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Help</string>
   </property>
   <property name="toolTip">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DeselectableQTreeView</class>
   <extends>QTreeView</extends>
   <header>varats/gui/deselectable_tree_view.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
